src/BinDeps.jl

```diff
@@ -182,7 +193,7 @@ module BinDeps
         description::String
         step::SynchronousStepCollection
         Choice(name,description,step) = (s=SynchronousStepCollection();lower(step,s);new(name,description,s))
-    end 
```

Comment:
It would make the patch much easier to read if you put all of your whitespace deletions in a separate PR.